### PR TITLE
Prepare for jdk 11

### PR DESCRIPTION
**Pull request by mbaechler**

### CODE REVIEW

**Comment by rouazana** [comment]
```
[e9896860f043336e46836066201d72f7ff7875d0] [ERROR] Errors: 
[e9896860f043336e46836066201d72f7ff7875d0] [ERROR]   JamesMailSpoolerTest.threadSuicideShouldAckTheItem:126 » ConditionTimeout Cond...
```
test this please (and should be the next flacky test to dig in)

File: pom.xml [hunk 1]
```diff
@@ -2156,6 +2177,21 @@
...
+                <artifactId>javax.annotation-api</artifactId>
+                <version>1.3.2</version>
+            </dependency>
+            <dependency>
+                <groupId>javax.xml.bind</groupId>
+                <artifactId>jaxb-api</artifactId>
+                <version>${jaxb.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>javax.annotation</groupId>
```
Comment (mbaechler) [inline pom.xml]: auto-review : looks duplicated

File: server/container/metrics/metrics-es-reporter/pom.xml [hunk 2]
```diff
@@ -80,6 +80,10 @@
             <groupId>javax.inject</groupId>
             <artifactId>javax.inject</artifactId>
         </dependency>
+        <dependency>
```
Comment (mbaechler) [inline server/container/metrics/metrics-es-reporter/pom.xml]: bad ordering

File: metrics/metrics-dropwizard/pom.xml [hunk 3]
```diff
@@ -50,6 +50,10 @@
             <groupId>javax.inject</groupId>
             <artifactId>javax.inject</artifactId>
         </dependency>
+        <dependency>
```
Comment (mbaechler) [inline metrics/metrics-dropwizard/pom.xml]: bad ordering

File: javax-mail-extension/pom.xml [hunk 4]
```diff
@@ -36,6 +36,10 @@
             <groupId>com.sun.mail</groupId>
             <artifactId>javax.mail</artifactId>
         </dependency>
+        <dependency>
+            <groupId>javax.activation</groupId>
```
Comment (mbaechler) [inline javax-mail-extension/pom.xml]: bad ordering

File: backends-common/rabbitmq/pom.xml [hunk 5]
```diff
@@ -81,6 +81,10 @@
             <groupId>javax.inject</groupId>
             <artifactId>javax.inject</artifactId>
         </dependency>
+        <dependency>
```
Comment (mbaechler) [inline backends-common/rabbitmq/pom.xml]: bad ordering

**Comment by mbaechler** [comment]
can be considered green, it failed after some deployment tests success

**Comment by mbaechler** [comment]
merged

**Pull request closed without merging**